src/WebJobs.Script/Workers/ProcessManagement/WorkerDescription.cs

```diff
@@ -45,6 +45,11 @@ public abstract class WorkerDescription
         /// </summary>
         public abstract bool UseStdErrorStreamForErrorsOnly { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the worker description is disabled.
+        /// </summary>
+        public bool? IsDisabled { get; set; }
+
```

Comment:
FYI - This allows both enabling and disabling.

 1 .  The parent level description can have `IsDisabled` set to True and child profile can override that to `False`
 2 .  The parent level description can have `IsDisabled` set to False and child profile can override that to `True`